lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -328,8 +328,9 @@ export abstract class ClientApplication {
      * @param state
      */
     private async handleHash(hash: string, state: string): Promise<AuthenticationResult> {
-        this.logger.verbose("handleHash called");
         const cachedRequest = this.browserStorage.getCachedRequest(state, this.browserCrypto);
+        const browserRequestLogger = new Logger({...this.config.system.loggerOptions, correlationId: cachedRequest.correlationId}, name, version);
```

Comment:
In order to prevent breaking changes to people calling the `setLogger` method we should probably call `this.logger.clone` with the new correlationId, etc. here

Reply:
Makes sense. Updated.